sw/device/tests/power_virus_systemtest.c

```diff
@@ -1454,6 +1454,48 @@ bool test_main(void) {
   if (kDeviceType == kDeviceSimDV || kDeviceType == kDeviceSimVerilator) {
     configure_pinmux_sim();
   }
+
```

Comment:
These should probably be moved to be inside the `configure_pinmux_sim()` function.